test/JDBC/expected/login_token.out

```diff
@@ -8,6 +8,12 @@ go
 create user login_token2;
 go
 
+create role login_token_role
+go
+
+alter role login_token_role add member login_token2
+go
+
```

Comment:
Add negative test case for server role membership i.e. sysadmin membership.
Please add tests for Windows Group Membership.